### PR TITLE
Display error on credential storage failure.

### LINKED ISSUE #6: Failure during credential claim/storage can result in `rejected` credential at the issuer
If the credential claim/acceptance/storage process does not complete properly, or is aborted part way through, the credential can be marked as 'rejected' in the issuer database.

### PR DESCRIPTION
**Pull request by mattcollier**
Fixes #6

### CODE REVIEW

File: accept-credential-component.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * @author Dave Longley
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.component('brAcceptCredential', {
+    bindings: {
+      onComplete: '&brAcceptCredentialCallback',
```
Comment (dlongley) [inline accept-credential-component.js]: Can this be called `brOnAccept` instead?

Comment (mattcollier) [inline accept-credential-component.js]: It's used for accept and reject actions.

Comment (dlongley) [inline accept-credential-component.js]: If too much overhead to fix now, add a TODO to change the naming at least.

Comment (dlongley) [inline accept-credential-component.js]: I'm mostly going for something that uses a consistent naming convention: `brOnSomething` vs. `brSomethingCallback`. We should use the former, not the latter.

**Comment by dlongley** [comment]
Just one comment to address, then LGTM, merge.

**Pull request merged**